src/js/rivet-collapsible.js

```diff
@@ -151,20 +151,28 @@
       switch (event.keyCode) {
         // Up arrow key
         case keys.up:
-          // If the current active toggle is the first bail
-          if (accordionTogglesArr[prevToggle] === undefined) return;
-
-          // Focus the previous toggle
-          accordionTogglesArr[prevToggle].focus();
+          if (!event.target.hasAttribute(DATA_ATTR)) {
```

Comment:
Looks like this code fixes the bug, but you might think about a small bit of refactoring. For example, you could clean up the this `if () else {}` statement here by combining the nested `if (accordionTogglesArr[prevToggle] === undefined) return;` statement below into a one-liner short circuit condition inside the first parent `if()`.

We want to kill execution of the script if _either_ of those conditions are true, so the `||` operator would be a good choice here.

```js
if (!event.target.hasAttribute(DATA_ATTR) || accordionTogglesArr[nextToggle] === undefined) {
  return;
} else {
  // Focus the next toggle
  accordionTogglesArr[nextToggle].focus();
}
```